backend/services/src/setup/handler.ts

```diff
@@ -213,6 +216,8 @@ export const handler: Handler = async (event) => {
     }
   });
 
+  configurationSettingsService.saveSetting(ConfigurationSettingsType.isTransferFrozen, false);
```

Comment:
What if we does not save this. What is the default behaviour? If this does not save, behaviour should be false

Reply:
removed from setup handler